explorer/src/components/account/AnchorAccountCard.tsx

```diff
@@ -0,0 +1,176 @@
+import React from "react";
+
+import { Account } from "providers/accounts";
+import { Address } from "components/common/Address";
+import { BorshAccountsCoder } from "@project-serum/anchor";
+import { capitalizeFirstLetter } from "utils/anchor";
+import { ErrorCard } from "components/common/ErrorCard";
+import { PublicKey } from "@solana/web3.js";
+import BN from "bn.js";
+
+import ReactJson from "react-json-view";
+import { useCluster } from "providers/cluster";
+import { useAnchorProgram } from "providers/anchor";
+
+export function AnchorAccountCard({ account }: { account: Account }) {
+  const { url } = useCluster();
+  const program = useAnchorProgram(
+    account.details?.owner.toString() ?? "",
+    url
+  );
+
+  if (!program) {
+    return (
+      <ErrorCard text={"Could not decode anchor program for this address"} />
```

Comment:
Nit: if the value is just a string, you can just…
```suggestion
      <ErrorCard text="Could not decode anchor program for this address" />
```

Reply:
Fixed in latest commit